services/common-features.yaml

```diff
@@ -81,3 +82,11 @@ features:
     title: Versioning
     description: |
       Providing the ability to maintain multiple versions of the same resource.
+  - id: CCC.F19
+    title: On Demand Scaling
```

Comment:
I'm not sure whether I prefer "On-Demand Scaling" or "Scaling On Demand" but I know I do not like "On Demand Scaling"! Yes, you can call me pernickety if you'd like, so I'll leave the final choice to you!